bin/citgm-all.js

```diff
@@ -23,6 +23,7 @@ yargs = commonArgs(yargs)
   });
 
 var app = yargs.argv;
+process.setMaxListeners(process.getMaxListeners() + app.parallel || process.getMaxListeners());
```

Comment:
I think this needs to happen down at line 59 or `app.parallel` will not exist yet